client/app/scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -40,13 +40,13 @@
 
 <div class="field">
     <label translate>Dateline</label>
-    <div sd-meta-locators class="dateline-info" ng-show="_editable && !proofread"
+    <div sd-meta-locators class="dateline-info" ng-show="_editable"
```

Comment:
OK. If that's meant for theme toggling then can we have different name other than proof read? I thought proof read is meant for article proof reading.